Dockerfile

```diff
@@ -12,7 +13,7 @@ RUN apt-get update && apt-get install -y \
     lcov valgrind vim \
     && pip3 install uwsgi
 
-# install node 12.x
+# install node 12.x (only for taccsite_custom)
 RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
```

Comment:
**Bonus.** Update Node version to latest v17 release.

Reply:
Calm down, Wes.

See this done via [FP-1531](https://jira.tacc.utexas.edu/browse/FP-1531): Update Node (from 12 to LTS or Latest), instead.